cuegui/cuegui/MenuActions.py

```diff
@@ -466,40 +484,78 @@ def eatDead(self, rpcObjects=None):
+                            ", ".join(blocked_job_owners)))
                 self._update()
 
     autoEatOn_info = ["Enable auto eating", None, "eat"]
 
     def autoEatOn(self, rpcObjects=None):
         jobs = self._getOnlyJobObjects(rpcObjects)
         if jobs:
+            blocked_job_owners = []
             for job in jobs:
-                job.setAutoEat(True)
-                job.eatFrames(state=[opencue.compiled_proto.job_pb2.DEAD])
+                if not cuegui.Utils.isPermissible(job):
+                    blocked_job_owners.append(job.username())
+                else:
+                    job.setAutoEat(True)
+                    job.eatFrames(state=[opencue.compiled_proto.job_pb2.DEAD])
+            if blocked_job_owners:
+                cuegui.Utils.showErrorMessageBox(
+                    AbstractActions.USER_INTERACTION_PERMISSIONS.format(
+                        "enable auto eating frames",
+                        ", ".join(blocked_job_owners)))
             self._update()
 
     autoEatOff_info = ["Disable auto eating", None, "eat"]
 
     def autoEatOff(self, rpcObjects=None):
         jobs = self._getOnlyJobObjects(rpcObjects)
         if jobs:
+            blocked_job_owners = []
             for job in jobs:
-                job.setAutoEat(False)
+                if not cuegui.Utils.isPermissible(job):
+                    blocked_job_owners.append(job.username())
+                else:
+                    job.setAutoEat(False)
+            if blocked_job_owners:
+                cuegui.Utils.showErrorMessageBox(
+                    AbstractActions.USER_INTERACTION_PERMISSIONS.format(
+                        "disable auto eating frames",
+                        ", ".join(blocked_job_owners)))
             self._update()
 
     retryDead_info = ["Retry dead frames", None, "retry"]
 
     def retryDead(self, rpcObjects=None):
         jobs = self._getOnlyJobObjects(rpcObjects)
         if jobs:
+            # check permissions
             if cuegui.Utils.questionBoxYesNo(self._caller, "Confirm",
                                              "Retry all DEAD frames in selected jobs?",
                                              [job.data.name for job in jobs]):
+                blocked_job_owners = []
                 for job in jobs:
-                    job.retryFrames(
-                        state=[opencue.compiled_proto.job_pb2.DEAD])
+                    if not cuegui.Utils.isPermissible(job):
+                        blocked_job_owners.append(job.username())
+                    else:
+                        job.retryFrames(
+                            state=[opencue.compiled_proto.job_pb2.DEAD])
+                if blocked_job_owners:
+                    cuegui.Utils.showErrorMessageBox(
+                        AbstractActions.USER_INTERACTION_PERMISSIONS.format(
+                            "retry dead for some of the selected jobs",
```

Comment:
retry dead **frames**:

"retry dead for some of the selected jobs" => "retry dead frames for some of the selected jobs"